bench/packw-benchmark.h

```diff
@@ -910,3 +978,44 @@ BENCHMARK_BGEMM(x32_packw_x8_gio__reference)
 BENCHMARK_BGEMM(x32_packw_x16_gio__reference)
 BENCHMARK_BGEMM(x32_packw_x32_gio__reference)
 BENCHMARK_BGEMM(x32_packw_x64_gio__reference)
+
+static void qb4_packw_goi__reference(
```

Comment:
consider gio reference, which is normally easy, but c4/c8 make gio harder than goi in some ways.
a scalar gio would be a good step before doing neon.

Reply:
yea i considered doing gio, but it seemed a lot more difficult for my ROI (i don't think i have any use case with transposed weights ATM). That might come back later for me to eventually finish, but ok with doing goi for now.